### PR TITLE
feat:predefinedEquipmentProperties of example for development

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: docker-compose/apps-metadata.json [hunk 1]
```diff
@@ -16,6 +16,13 @@
       "path" : "/studies/{elementUuid}"
     }
   ],
+  "predefinedEquipmentProperties" : {
+    "substation" : {
+      "region" : ["east", "south", "west", "north"],
+      "planet" : ["earth", "mars"],
```
Comment (etiennehomer) [inline docker-compose/apps-metadata.json:22]: Remove planet maybe

File: docker-compose/apps-metadata.json [hunk 2]
```diff
@@ -16,6 +16,13 @@
...
+      "totallyFree" : []
```
Comment (etiennehomer) [inline docker-compose/apps-metadata.json:23]: Put something more simple like "Category" ?

**Comment by etiennehomer** [comment]
Code ok, tests ok

File: docker-compose/apps-metadata.json (hunk 1 above)
Comment (laurgarn) [reply docker-compose/apps-metadata.json]: Done

File: docker-compose/apps-metadata.json (hunk 2 above)
Comment (laurgarn) [reply docker-compose/apps-metadata.json:31]: Added TSOs

**Pull request merged**